samples/WOCCatalog/WOCCatalog.vsimporter/WOCCatalog-WinStore10/WOCCatalog.vcxproj

```diff
@@ -88,12 +89,13 @@
       <HeaderMap>Project</HeaderMap>
       <IncludePaths>$(SolutionPublicHeadersDir)</IncludePaths>
```

Comment:
yeah probably but this is fine for now. Just trying to get builds working. We can re-re-import later if we want.

---
In reply to: [102805999](https://github.com/Microsoft/WinObjC/pull/2056#discussion_r102805999) [](ancestors = 102805999)